plugins/uniques.js

```diff
@@ -1,9 +1,8 @@
 // @author         3ch01c
 // @name           Uniques
 // @category       Misc
-// @version        0.2.4
-// @description    Allow manual entry of portals visited/captured. Use the 'highlighter-uniques' plugin to show the uniques on the map, and 'sync' to share between multiple browsers or desktop/mobile. It will try and guess which portals you have captured from COMM/portal details, but this will not catch every case.
-
+// @version        0.4.0
```

Comment:
0.3 was some intermediate release and I want to ensure that people get the update from release. so 0.4.0 was never used in either a draft or beta or RC and it will update.

Reply:
> 0.3 was some intermediate release

As I can see we never had such release on official site.

>  I want to ensure that people get the update from release.

Do you mean auto-update routines?
They also rely on `@updateURL`, so it's not enough to change version.